playbooks/roles/openconnect/templates/client.mobileconfig.j2

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>PayloadContent</key>
+	<array>
+		<dict>
```

Comment:
I think the indentation of this XML file went a little haywire.

Reply:
hmm seems right to me?

Reply:
Here's what it looks like after going through `xmllint --format`: https://gist.github.com/cpu/6a52c050c8df4b557b791d912e61e6df

In yours the indent on the `<dict>` for the initial payload content doesn't line up correctly and a few elements seem shifted.

Reply:
(The xmllinted copy isn't perfect either because I didn't remove the templating strings so a couple <data> are weird. It's close!)